magento/magento-cloud-patches
# MCLOUD-6683: BUNDLE-2670: Braintree VBE - Virtual Terminal Page errors and will not load

### Description:
https://jira.corp.magento.com/browse/MCLOUD-6683:
Adds patch to fix issue for Braintree VBE where Braintree Virtual Terminal page throws console errors and displays loading spinner forever

### Fixed Issues:
https://jira.corp.magento.com/browse/BUNDLE-2670

### Manual Testing Scenarios
1. Enable Braintree
2. Navigate to: Sales > Braintree Virtual Terminal
3. Page should load correctly without errors
